src/common/tusb_fifo.c

```diff
@@ -923,103 +911,90 @@ uint16_t tu_fifo_get_linear_read_info(tu_fifo_t *f, uint16_t offset, void **ptr,
   }
 
   // Skip beginning of buffer
```

Comment:
`Skip beginning of buffer` does not seem to fit anymore. It should be `Check If the fifo is empty`.